lib/ansible/config/base.yml

```diff
@@ -414,17 +414,6 @@ ACTION_WARNINGS:
   - {key: action_warnings, section: defaults}
   type: boolean
   version_added: "2.5"
-COMMAND_WARNINGS:
-  name: Command module warnings
-  default: True
-  description:
-    - By default Ansible will issue a warning when the shell or command module is used and the command appears to be similar to an existing Ansible module.
-    - These warnings can be silenced by adjusting this setting to False. You can also control this at the task level with the module option ``warn``.
-  env: [{name: ANSIBLE_COMMAND_WARNINGS}]
-  ini:
-  - {key: command_warnings, section: defaults}
-  type: boolean
-  version_added: "1.8"
```

Comment:
Change the default to `False` here and add a deprecation key (look around for examples).